### ISSUE TITLE
[Hunter] Traps not working anymore

### ISSUE THREAD

**Issue by Somehunter** [dalaranwow/dalaran-wow#4921]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
- When i lay down a Hunter Trap, doesnt matter wich one, it appears normaly. When it should be triggered then by an enemy target it just disappears. no slow, dmg, snakes, beacons etc

**Expected behaviour**: Tell us what should happen instead
- What traps normaly do

**Steps to reproduce the problem**:

1. Play Hunter
2. Lay down a trap and pull an enemy target
3. Watch

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Lothloryen** [comment]
Hello there,

Try them now, after this maintenance they should be working again.

Thank you  for your report.

**Comment by Somehunter** [comment]
i just did. not working sorry

**Comment by Lothloryen** [comment]
Hello @Somehunter , you are right. It will be fine after the next server restart.
